config/initializers/spotlight_initializer.rb

```diff
@@ -13,7 +13,7 @@
     label: -> { I18n.t(:'spotlight.search.fields.spotlight_upload_attribution_tesim') }
   ),
   Spotlight::UploadFieldConfig.new(
-    field_name: %w(date spotlight_upload_date_tesim date_sort),
```

Comment:
Do we need to support the `date` field still?

Reply:
No, because its not a real solr field.